src/backend/vector/avx2/edwards.rs

```diff
@@ -544,4 +544,32 @@ mod test {
             assert_eq!(base_splits[3], b_splits[3]);
         }
     }
+
+    #[test]
+    fn b_shl_128_odd_lookup_table_verify() {
+        use crate::backend::vector::avx2::constants::B_SHL_128_ODD_LOOKUP_TABLE;
+        use crate::constants;
+        use crate::scalar::Scalar;
+
+        let b_shl_128_odd_table = NafLookupTable8::<CachedPoint>::from(
+            &(constants::ED25519_BASEPOINT_POINT
+                * Scalar::from_canonical_bytes([
+                    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                    0, 0, 0, 0, 0, 0,
+                ])
+                .unwrap()),
+        );
+        println!("b_shl_128_odd_lookup_table = {:?}", b_shl_128_odd_table);
```

Comment:
I added this test to match the one for `BASEPOINT_ODD_LOOKUP_TABLE`, and used it to regenerate the AVX2 `B_SHL_128_ODD_LOOKUP_TABLE` table (the contents of which apparently gets generated differently after over 2 years of crate development, but both the old and new lookup tables pass tests).